lib/generators/paper_trail/install_generator.rb

```diff
@@ -31,6 +32,13 @@ def create_migration_file
       end
     end
 
+    def create_initializer
+      create_file(
```

Comment:
The system outputs a deprecation warning is you don't explicitly specify false.  Therefore, all new users will see a deprecation warning if this initializer isn't generated for `false` by default, which I believe was the original goal of #830 that this PR was meant to solve.   Handling the `true` case was extra.